### DISCUSSION TITLE
为设备列表里面，添加到地址薄的时候，没有选择地址薄的选项，只有一个标签的选项，

### DISCUSSION THREAD

**Discussion by rocken1990** [lejianwen/rustdesk-api#84] [General]
选择了标签，设备会直接在我的地址薄上重新生成一个标签，而不是在创建好的地址薄上

**Comment by lejianwen** [comment]
> 选择了标签，设备会直接在我的地址薄上重新生成一个标签，而不是在创建好的地址薄上

这确实是个bug，我改一下

> 为设备列表里面，添加到地址薄的时候，没有选择地址薄的选项，只有一个标签的选项

这个是因为可以一次添加选多个人，所以没办法选地址簿了
